warpcast-langchain/warpcast_toolkit.py

```diff
@@ -0,0 +1,50 @@
+from langchain_core.tools import BaseTool
+from langchain_core.tools.base import BaseToolkit
+
+from cdp_agentkit_core.actions.social.warpcast import (
+    CastAction,
+    ReplyAction,
+    UserDetailsAction,
+    UserCastsAction,
+)
+from warpcast_langchain.warpcast_api_wrapper import WarpcastApiWrapper
+
+
+class WarpcastToolkit(BaseToolkit):
+    """Warpcast Toolkit for Langchain integration."""
+
+    tools: list[BaseTool] = []  # noqa: RUF012
+
+    @classmethod
+    def from_warpcast_api_wrapper(cls, warpcast_api_wrapper: WarpcastApiWrapper) -> "WarpcastToolkit":
+        """Create a WarpcastToolkit from a WarpcastApiWrapper.
+
+        Args:
+            warpcast_api_wrapper: WarpcastApiWrapper. The Warpcast API wrapper.
+
+        Returns:
+            WarpcastToolkit. The Warpcast toolkit.
+
+        """
+        actions = [
```

Comment:
Please follow twitter toolkit's existing [pattern](https://github.com/coinbase/cdp-agentkit/blob/12ddc1175474f4b8a79aeff6d0fb3e51fb4c9aa2/twitter-langchain/twitter_langchain/twitter_toolkit.py#L123) to import and initialize all actions available from [cdp-agentkit-core](https://github.com/coinbase/cdp-agentkit/blob/12ddc1175474f4b8a79aeff6d0fb3e51fb4c9aa2/cdp-agentkit-core/cdp_agentkit_core/actions/social/twitter/__init__.py#L8)